core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdminTestBase.java

```diff
@@ -723,4 +723,61 @@ public void upgrade_ShouldCallJdbcAdminProperly() throws ExecutionException {
     // Arrange
     verify(distributedStorageAdmin).upgrade(options);
   }
+
+  @Test
+  public void coordinatorNamespaceShouldHandleCorrectly() throws ExecutionException {
+    // Arrange
+    when(admin.getNamespaceNames())
```

Comment:
```suggestion
    when(distributedStorageAdmin.getNamespaceNames())
```
I might be missing something, but it looks the storage admin needs to be used.

Reply:
Ah, that's correct. But why does the current test code work...? Anyway, I will fix it. Thanks.

Reply:
Fixed in c1161422ac6e7deb6460ffd8523ad197fe4b6698. Thank you!